browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -6434,7 +6436,7 @@ index 6a95f2aa052feeb07e38a7192d02d98468f7e5cb..b36ed11386c28cefc844901a0b57c45e
 +    if (element.isMouseFocusable())
 +        return true;
 +
-+    if (auto shadowRoot = makeRefPtr(element.shadowRoot())) {
++    if (RefPtr shadowRoot = element.shadowRoot()) {
```

Comment:
Does `RefPtr` without template parameter compile?

Reply:
Yes, it builds. I built with the changes applied on Ubuntu 20.04 and Ubuntu 18.04. So it should build on both systems.

Regarding the change, I took a look at the changes applied in the set of patches that removed makeRef/makeRefPtr. Here's a very similar change: https://github.com/WebKit/webkit/compare/102a6d5be26fa%5E..102a6d5be26fa#diff-37167271e633e42f6b0cb277c8eb74680c5515cb6eef47bb4321897e5f5314e9L400 from bug Drop remaining uses of makeRefPtr() in WebCore/ https://bugs.webkit.org/show_bug.cgi?id=230527